src/app/search/search.component.spec.ts

```diff
@@ -1,25 +1,34 @@
+import { AdvancedSearchStubService } from './../test/service-stubs';
+import { AdvancedSearchService } from './advancedsearch/advanced-search.service';
+import { SearchStubService } from '../test/service-stubs';
+import { SearchService } from './search.service';
+import { ProviderService } from './../shared/provider.service';
+import { NO_ERRORS_SCHEMA } from '@angular/core';
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
 
 import { SearchComponent } from './search.component';
 
 describe('SearchComponent', () => {
-  let component: SearchComponent;
```

Comment:
This file seems totally commented out, does it do anything anymore?

Reply:
The tests weren't passing and Jennifer's changes will probably modify the search component further.  Leaving this test for later.  My misc ui2 bugs issue indirectly references it.